spec/10-overview.md

```diff
@@ -1,16 +1,13 @@
 # Overview
 
-Trace context consists of a set of properties crucial for event correlation in distributed applications. Trace context positions requests in a <a>distributed trace</a>. If you look at a <a>distributed trace<a> as a graph of correlated requests - a single request can be part of multiple graphs. For example, let's say component `A` calls component `B` which in turn calls `C`. One distributed tracing system may see component `A` calling component `B`. Another may see all three components communicating. So the call from `B` to `C` will carry trace context about service `A` as well as information about its direct communication upstream to `B`.
+Trace context is split into two individual propagation fields supporting interoperability and vendor-specifc extensability:
 
-The situation when a single request is a part of multiple graphs is becoming more common. One distributed application may have components monitored by different distributed tracing systems. Those distributed tracing systems may not be easily replaceable as they might be provided by cloud vendors or distributed as pre-built components.
+- `traceparent` describes the position of the incoming request in its trace graph in a portable, fixed-length format. Its design focuses on fast parsing. Every tracing tool MUST properly set `tracestate` even when it only relies on vendor-specif informaiton in `tracestate`
+- `tracestate` extends `traceparent` with vendor-specific data represented by a set of name/value pairs. Storing information in `tracestate` is optional.
 
-The trace context is designed to allow extensibility for all distributed tracing systems. It requires them to respect context set by other systems.
+Tracing tools can provide two levels of compliant behaviour interacting with trace context:
 
-Trace context is represented by a set of name/value pairs describing the identity of every http request. Two propagation fields carry the common and vendor-specific properties that make up the trace context.
+- At a minimum they MUST propagate the `traceparent` and `tracestate` headers and guarantee  traces are not broken. this behaviour is also referred to as forwarding a trace.

```

Comment:
Formatting: there's an extra space between "guarantee" and "traces"

Reply:
Fixed